common/beerocks/bcl/source/beerocks_socket_thread.cpp

```diff
@@ -262,6 +262,15 @@ bool socket_thread::work()
 {
     before_select();
 
+    auto awake_time_msec = std::chrono::duration_cast<std::chrono::milliseconds>(
+                               std::chrono::steady_clock::now() - m_select_wake_up_time)
+                               .count();
+
+    if (m_select_timeout_msec > 0 && awake_time_msec > m_select_timeout_msec) {
+        THREAD_LOG(WARNING) << "Thread awake time is exceptionally long: " << int(awake_time_msec)
+                            << " [ms]";
```

Comment:
```suggestion
        THREAD_LOG(WARNING) << "Thread awake time is exceptionally long: " << int(awake_time_msec)
                            << "/" << m_select_timeout_msec << " [ms]";
```

Reply:
I prefer not. The select timeout is not interesting IMO. The user will understand what's wrong.

Reply:
It's **very** interesting because you're using it to calculate the "allowed" awake timeout.